doc/man1/openssl-verification-options.pod

```diff
@@ -548,10 +569,14 @@ This option has no effect and is retained for backward compatibility only.
 
 =head2 Certificate Extensions
 
-Options like B<-purpose> lead to checking the certificate extensions,
+Options like B<-purpose> lead to checking specific certificate extensions,
```

Comment:
```suggestion
The B<-purpose> options triggers the processing of specific certificate extensions,
```

Reply:
Changed to:
`Options like B<-purpose> and B<-verify_name> trigger the processing of`